packages/admin/admin/src/form/FinalFormFileSelect.tsx

```diff
@@ -22,115 +22,235 @@ export type FinalFormFileSelectClassKey =
     | "fileListItemInfos"
     | "rejectedFileListItem"
     | "errorMessage"
-    | "droppableAreaIsDisabled"
-    | "droppableAreaHasError"
```

Comment:
These keys are still used in the owner state, so they should probably not be removed.